packaging_scripts/assemble.sh

```diff
@@ -390,7 +390,7 @@ function main() {
     echo "Assembling wazuh-indexer for $PLATFORM-$DISTRIBUTION-$ARCHITECTURE"
 
     VERSION=$(bash packaging_scripts/upstream_version.sh)
-    ARTIFACT_BUILD_NAME=$(ls "${OUTPUT}/dist/" | grep "wazuh-indexer-min.*$SUFFIX.*\.$EXT")
+    ARTIFACT_BUILD_NAME=$(ls "${OUTPUT}/dist/" | grep "wazuh-indexer-min-$(<VERSION).*$SUFFIX.*\.$EXT")
```

Comment:
grep expression does not match our packages' naming convention:

**Development package**
```
any ~~ <SubsystemName>_<VersionNumber>-<Revision>_<Architecture>_<GitRef>.<PackageType>
```


**Release (stage) package**
```
deb ~~ <SubsystemName>_<VersionNumber>-<Revision>_<Architecture>.<PackageType>
rpm ~~ <SubsystemName>_<VersionNumber>-<Revision>.<Architecture>.<PackageType>
```
This section is very sensible, do not add changes if not necessary.

Reply:
https://github.com/wazuh/wazuh-indexer/actions/runs/10900242826

Reply:
This commit was rolled back